.github/workflows/wheels.yml

```diff
@@ -0,0 +1,36 @@
+name: Wheels
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+  push:
+    branches:
+      - main
```

Comment:
I think this file makes sense to me. When we are ready to cut a release binary against PyTorch 1.11, would we create a separate functorch branch (e.g. release/0.1.0) and modify this file to install the PyTorch RC?

Reply:
Good point, let me add `- release/*` wildcard, and tags one as well